edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleIdentityLifecycleManagerTest.cs

```diff
@@ -237,5 +237,48 @@ public async Task TestGetModulesIdentity_WithUnchanged_ShouldReturnAllWhenReques
 
             Mock.Get(identityManager).Verify(im => im.DeleteIdentityAsync(Module3));
         }
+
+        [Fact]
+        [Unit]
+        public async Task TestGetModulesIdentity_WithBadModules_ShouldRemoveThoseIdentities()
```

Comment:
not: instead of WithBadModules maybe name it WithExistingIdentities or WithExtraIdentities.